avocado/plugins/replay.py

```diff
@@ -0,0 +1,96 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat, Inc. 2020
+# Authors: Cleber Rosa <crosa@redhat.com>
+
+"""Replay Job Plugin"""
+
+import json
+import os
+import sys
+
+from avocado.core.data_dir import get_job_results_dir
+from avocado.core.dispatcher import JobPrePostDispatcher
+from avocado.core.future.settings import settings
+from avocado.core import job, exit_codes, output
+from avocado.core.plugin_interfaces import CLICmd
+
+
+class Replay(CLICmd):
+
+    """Implements the avocado 'replay' subcommand."""
+
+    name = 'replay'
+    description = "Runs a new job using a previous job as its configuration"
+
+
+    def configure(self, parser):
+        parser = super(Replay, self).configure(parser)
+
+        help_msg = ('Replays a job, identified by: complete or partial Job '
+                    'ID, "latest" for the latest job, the job results path.')
+        settings.register_option(section='job.replay',
+                                 key='source_job_id',
+                                 help_msg=help_msg,
+                                 metavar='SOURCE JOB ID',
```

Comment:
would be nice if metavar would be one word, to avoid confusing during the command line syntax.